app/lib/research_data_harvester.rb

```diff
@@ -15,4 +16,28 @@ def collections_to_index
     end
     collections
   end
+
+  ##
+  # For a given ResearchDataCollection, retrieve its metadata from DataSpace
+  # @param [ResearchDataCollection] research_data_collection
+  def harvest(research_data_collection)
+    server = 'https://dataspace-dev.princeton.edu/rest'
+    collection_id = research_data_collection.collection_id
```

Comment:
question: Are you expecting to use other properties of the collection here? If not, maybe just pass the collection ID?

Reply:
Yes, we will need the other properties of collection when we're indexing, I just haven't written that part yet.